cypress/e2e/admin/administrateMigration.feature

```diff
@@ -0,0 +1,26 @@
+@release
+Feature: Admin Users - To administrate the Migration process.
+
+  As an Admin I want to start, finish and make the migration mandatory.
+
+  Background:
+    Given I am logged in as a 'admin' at 'nbc'
+    When I go to administration page
+    When I go to school administration
+
+  Scenario: Admin starts the migration and the School Number is added
+    Then I can see the school number
+    Then I see button Start migration is enabled
+    Then I can see the migration information text
+    Then I can write an email to a pre-defined receiver
+    Then I see the information link href is blog.niedersachsen.cloud/umzug
+    When I click on the start migration button
+    When I click on agree migration button
+    Then I see the migration is active field
+    Then I see the end migration button
+
+
+  Scenario: Admin ends the migration
+    When I click on end migration button
+    When I click on the end migration confirmation checkbox
+    When I click on the end migration confirmation button
```

Comment:
Is there 'Then' required to see what is expected after these actions from line #24 to 26?

Reply:
This test should be written as a part of another ticket, I added this just to make the test more suitable for multiple re runs, because not ending a migration runs problems. I will be expanded in the future. Hope it is ok like this for now.

Reply:
@HBukvic you could make a comment with the ticket number. We will do it in the next sprint @atlldwp

Reply:
https://ticketsystem.dbildungscloud.de/browse/N21-1095

Reply:
ok, thanks for clarifying.